app/frontend/src/submissions/components/SubmissionForm/Step06Casings.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <fieldset>
+    <legend>Casing Details</legend>
+    <table class="table">
```

Comment:
wrap table in `<div class="table-responsive">`? (or else it may spill out of the container)